lib/torque/mercator.js

```diff
@@ -85,7 +85,7 @@ MercatorProjection.prototype.latLonToTilePoint = function(lat, lon, tileX, tileY
   var worldCoordinate = this._fromLatLonToPoint(lat, lon);
   var pixelCoordinate = new Point(worldCoordinate.x*numTiles, worldCoordinate.y*numTiles);
   var tilePixelPos    = this._tilePixelPos(tileX, tileY);
-  return new Point(Math.round(pixelCoordinate.x-tilePixelPos.x), Math.round(pixelCoordinate.y-tilePixelPos.y));
+  return new Point(Math.round(pixelCoordinate.x-tilePixelPos.x), this._tileSize - Math.round(pixelCoordinate.y-tilePixelPos.y));
```

Comment:
No one appears to be using those functions, but I can always handle that on my provider if you prefer.